### PR TITLE
First step

### PR DESCRIPTION
**Pull request by antoine-de**
init the project with stuff copied from https://github.com/CanalTP/munin

now to import bano data into ES you can do:

```sh
./target/release/bano2mimir bano-data*/bano-*.csv
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "mimirsbrunn"
+version = "0.0.1"
+authors = ["Guillaume Pinot <texitoi@texitoi.eu>"]
```
Comment (TeXitoi) [inline Cargo.toml]: You should put Casio Numérique or something like that here.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+
 mimir data import
+
+## build
+
+To build, you must first install rust:
+```shell
+curl -s https://static.rust-lang.org/rustup.sh | sudo sh
+```
+and then build munin:
```
Comment (TeXitoi) [inline README.md]: mimirsbrunn?

File: src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (TeXitoi) [inline src/lib.rs]: lack licence

File: src/bin/bano2mimir.rs [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            street: street,
+            name: addr_name,
+            coord: mimirsbrunn::Coord { lat: self.lat, lon: self.lon },
+            weight: 1,
+        }
+    }
+}
+
+fn index_bano(files: &[String]) {
+    println!("purge and create Munin...");
```
Comment (pbougue) [inline src/bin/bano2mimir.rs]: Mimir (*3)

File: src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    pub id: String,
+    pub house_number: String,
+    pub street: Street,
+    pub name: String,
+    pub coord: Coord,
+    pub weight: u32,
+}
+
+pub fn purge_and_create_munin() -> Result<(), curl::ErrCode> {
+    // first, we must delete with its own handle the old munin
```
Comment (pbougue) [inline src/lib.rs]: mimir

File: README.md (hunk 2 above)
Comment (fdegomme-canaltp) [inline README.md]: Comment gère-t-on le packaging de rust ?

Comment (antoine-de) [inline README.md]: we'll do debian packages.

at the end it's static linked binaries, so it's pretty easy

Comment (fdegomme-canaltp) [inline README.md]: Well ! :)

**Pull request merged**